src/locale/locales/fr/messages.po

```diff
@@ -1745,11 +1663,11 @@ msgstr "Pseudo"
 
 #: src/components/RichText.tsx:188
 msgid "Hashtag: {tag}"
-msgstr ""
+msgstr "Hashtag: {tag}"
 
 #: src/view/com/auth/create/CreateAccount.tsx:208
 msgid "Having trouble?"
-msgstr ""
+msgstr "Vous avez des problèmes ?"
```

Comment:
So, yeah, well, _technically_, it could mean that, but I read it like _Do you have problems (in your life)?_, which, well, I mean, yeah, but, well, no, I don't think it's what we mean here :D

I would opt for something a little bit less intrusive yet is shorter:

```suggestion
msgstr "Un souci ?"
```